apps/web/README.md

```diff
@@ -12,16 +12,16 @@
 
 ### The desktop app?
 
-You can find all the desktop related code in [the `desktop/` directory](./desktop/). Since it uses the web app directly, we are keeping both together. (We should probably move it to it's own project at some point.)
+You can find all the desktop related code in the [desktop](./desktop/) directory. Since it uses the web app directly, we are keeping both together. (We should probably move it to it's own project at some point.)
 
 ### Downloads & releases
 
-- Web app: [try it out here](https://app.notesnook.com/)
-- Desktop app: [download for your platform from here](https://notesnook.com/downloads)
+- Web app: [Try it out here.](https://app.notesnook.com/)
+- Desktop app: [Download for your platform from here.](https://notesnook.com/downloads)
```

Comment:
Why are we capitalizing here?

Reply:
Its not a sentence in continuation. Its like web app and then try it out here. So, I capitalized them.